apps/src/aiTutor/views/AITutorContainer.tsx

```diff
@@ -0,0 +1,79 @@
+import classnames from 'classnames';
```

Comment:
<img width="1217" alt="Screenshot 2024-04-10 at 11 13 52 PM" src="https://github.com/code-dot-org/code-dot-org/assets/26844240/2a849b93-2b62-40af-b4b0-e9313e474670">
<img width="1221" alt="Screenshot 2024-04-10 at 11 14 04 PM" src="https://github.com/code-dot-org/code-dot-org/assets/26844240/f509e6b7-3fe1-4d8e-b8ac-b08e5b45238d">
<img width="1197" alt="Screenshot 2024-04-10 at 11 14 15 PM" src="https://github.com/code-dot-org/code-dot-org/assets/26844240/925d2928-8779-4115-b69d-b2e66b16127d">

Reply:
This component changed significantly to accommodate the new organization/UI. I also made it draggable because it was helpful for testing. I've pasted the diff between this and the old equivalent, which was `aiTutorPanel`: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/components/aiTutor/aiTutorPanel.tsx